### ISSUE TITLE
the original_size and target_size in the train process

### ISSUE THREAD

**Issue by CuddleSabe** [Fanghua-Yu/SUPIR#61]
Hi, great work
I want to repeat it in diffusers
in diffusers, what should the parameter original_size and target_size embedding be?
thanks

**Comment by Fanghua-Yu** [comment]
Hello, conditions for SDXL can be found here (`aesthetic_score` is useless for SDXL-base):
https://github.com/Fanghua-Yu/SUPIR/blob/bca9a727b8a756ddaad7f13401631b5fde9f7f66/SUPIR/models/SUPIR_model.py#L154-L157

**Comment by JUSTzhuzhu** [comment]
> Hi, great work I want to repeat it in diffusers in diffusers, what should the parameter original_size and target_size embedding be? thanks

Hello, have you implemented the training script？

**Comment by CuddleSabe** [comment]
> > Hi, great work I want to repeat it in diffusers in diffusers, what should the parameter original_size and target_size embedding be? thanks
> 
> Hello, have you implemented the training script？

yes, I trans it to diffusers framework and it can work. But it is the company's property, so i cant share it
